content/en/about-types.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Types
```

Comment:
You'll need to do two more things so that this page actually shows up on the website:

1. Add the page to an appropriate place in `data/toc.json`.
2. Run `gulp` to rebuild the HTML files for the website.

Reply:
Done